### ISSUE TITLE
Lowgrade intraepithelial neoplasia

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6853]
**Preferred term label:**
Lowgrade intraepithelial neoplasia

**Synonyms**
LGIN

**Definition (free text, please give PubMed ID)**
The last stage of precancerous lesion.
DOI: https://doi.org/10.1155/2020/7907195

**Parent term (use hpo.jax.org/app)**
Neoplasm

**Comment by pnrobinson** [comment]
@johannaest This is not a great HPO term because it can apply to any form of epithelial cancer. Can you provide more information?

**Comment by pnrobinson** [comment]
This is an attribute of a tumor and does not appear to be a good HPO term at this time. If we want to add terms such as this we will need to have a more formal oncology workshop.
